### ISSUE TITLE
TargetClick should be a button

### ISSUE THREAD

**Issue by rickyp-uber** [snowcoders/react-popover#2]
TargetClick should be a button since it's a click event for accessibility purposes

**Comment by k2snowman69** [comment]
That makes sense... Give me a few days but yea. Should also be a patch update as it shouldn't affect backcompat. Look forward to that :-)

**Comment by k2snowman69** [comment]
https://github.com/snowcoders/react-popover/commit/e332b79bcc12f4805235be0597dc37f9831b1df6
